Dockerfile.server

```diff
@@ -2,8 +2,10 @@
 
 FROM node:16
 
-RUN apt-get update
-RUN apt-get --yes install vim
+RUN apt-get update \
+    && apt-get install -y \
+    vim \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
we can also add `apt-get clean`

Reply:
I was under the impression that apt has been doing auto-clean for a little while now and its not needed any more ;) Anyways that diff is not part of this PR anymore anyways.